### ISSUE TITLE
Multiple contractors

### ISSUE THREAD

**Issue by michalskop** [michalskop/smlouvy.gov.cz#1]
How to solve the problem with multiple contractors in CSV?

(They are now in one column as json array.)

**Comment by kokes** [comment]
Já to řešim přes zvlášť tabulku, kterou mám napojednou jako 1:Many na master se seznamem smluv. Bude se pak líp agregovat a jinak inspektovat seznam smluvních stran. https://github.com/kokes/smlouvy/blob/master/tabulkuj.py#L44

(Btw pozor na to, že spousta smluv má jako smluvní stranu i samotného objednavatele.)

**Comment by michalskop** [comment]
Jo, koukal jsem - to byla (a je pro snad budoucí API) i moje záložní varianta, ale chtěl jsem to zatím takto nechat, jestli pro CSVčka (datapackages) nevyleze něco lepšího.
